.github/workflows/release.yml

```diff
@@ -87,4 +89,3 @@ jobs:
           ./.github/workflows/upload_asset.sh ./extra/completions/_alacritty $GITHUB_TOKEN
           ./.github/workflows/upload_asset.sh ./extra/linux/Alacritty.desktop $GITHUB_TOKEN
           ./.github/workflows/upload_asset.sh ./extra/alacritty.info $GITHUB_TOKEN
-          ./.github/workflows/upload_asset.sh ./alacritty.yml $GITHUB_TOKEN
```

Comment:
Hello, my I know if there will be an alternative way to get the default TOML config with comments?

Reply:
Ok it's already been asked: https://github.com/alacritty/alacritty/issues/6592#issuecomment-1603702638